### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by ClareRadtke**
Hi Team,

Please review the folder structure and README doc.
Still a WIP as we will be adding to the README throughout the project.

### CODE REVIEW

**Comment by lewy192** [comment]
sounds good

**Comment by Honey-ant** [comment]
Great work! Appreciate your leadership :)

**Review by lewy192 — APPROVED** [review]
all good here

File: README.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+
+### Notes for Submission on BCS  
+
+Each member is required to submit the following:
+
+* The URL of the deployed application.
+
+* The URL of the GitHub repository.
 
-[FlowerPot](https://clareradtke.github.io/FlowerPot/)
```
Comment (Honey-ant) [inline README.md:3]: Fabulous

**Review by Honey-ant — COMMENTED** [review]
Great one liners

**Review by Honey-ant — APPROVED** [review]

**Pull request merged**